runtime/onert/core/src/compiler/ShapeValidator.cc

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Why is this copyright added?

Reply:
See https://github.com/Samsung/ONE/pull/15164#discussion_r2047906033. I've translated the logic from the code from:

```cc
int extending_dims = output_num_dims - input_num_dims;
for (int idx = 0; idx < input_num_dims; ++idx) {
  TF_LITE_ENSURE_MSG(context,
                     (SizeOfDimension(op_context->input, idx) == 1 ||
                      SizeOfDimension(op_context->input, idx) ==
                          get_shape_data(extending_dims + idx)),
                     "Output shape must be broadcastable from input shape.");
```